core/web/presenters/eth_key.go

```diff
@@ -49,7 +48,10 @@ func NewETHKeyResource(k models.Key, opts ...NewETHKeyOption) (*ETHKeyResource,
 		IsFunding:   k.IsFunding,
 		CreatedAt:   k.CreatedAt,
 		UpdatedAt:   k.UpdatedAt,
-		DeletedAt:   k.DeletedAt,
+	}
+
+	if k.DeletedAt.Valid {
```

Comment:
Is this still relevant? Calling Valid would make more sense on a `null.Time`

Reply:
It's not really relevant but I didn't want to make any changes to the CLI output

Reply:
Oops my mistake I thought this was a `ETHKeyResource` but its a `models.Key`. This is fine.